doc/sailfish-webview-webview.qdoc

```diff
@@ -41,6 +39,18 @@
   \sa WebVewPage
 */
 
+/*!
+  \qmlproperty bool active
+  \brief This property is \c{true} if the WebView is currently active.
+
+  The active property follows application visibility and status of the WebViewPage unless explicitly set.
+  When the application or WebView is hidden (not visible) for the end user, active should be false.
+
+  Touching the content forces WebView to active state.
+
+  Normally application developer do not need to alter the default behaviour.
```

Comment:
Just a really minor typo, but "application developers" (plural). Also, I'm not sure what the rule is, but I think elsewhere we use US-English (e.g. "initialize", "color"), in which case "behavior".

Reply:
US-English. Will fix.